test/runtests.jl

```diff
@@ -60,6 +60,8 @@ using TimeZones
         @test isnothing(JuliaCon.today())
         @test isnothing(JuliaCon.today(track="BoF"))
         @test isnothing(JuliaCon.today(terminal_links=true))
+        @test isnothing(JuliaCon.tomorrow())
+        @test isnothing(JuliaCon.tomorrow())
```

Comment:
Calling this once is enough, isn't it? 😄

Reply:
I was just doing what `today` had above. But I didn't understand why, so if you don't think it's needed. Removed

Reply:
Oh, I might have added it to test both the downloading/caching path and the "load from cache" path. In any case, unnecessary here.